src/auth/__tests__/AuthProvider.unit.test.tsx

```diff
@@ -33,51 +40,45 @@ describe("AuthProvider", () => {
   });
 
   it("provides access to all fields in interface (sanity check)", () => {
-    expect("user"            in auth).toBe(true);
-    expect("csrfToken"       in auth).toBe(true);
-    expect("check"           in auth).toBe(true);
+    expect("user" in auth).toBe(true);
+    expect("csrfToken" in auth).toBe(true);
+    expect("check" in auth).toBe(true);
     expect("isAuthenticated" in auth).toBe(true);
-    expect("isEditor"        in auth).toBe(true);
-    expect("post"            in auth).toBe(true);
-    expect("put"             in auth).toBe(true);
-    expect("patch"           in auth).toBe(true);
-    expect("delete"          in auth).toBe(true);
-    expect("setUser"         in auth).toBe(true);
-    expect("login"           in auth).toBe(true);
-    expect("logout"          in auth).toBe(true);
+    expect("isEditor" in auth).toBe(true);
+    expect("setUser" in auth).toBe(true);
+    expect("login" in auth).toBe(true);
+    expect("logout" in auth).toBe(true);
   });
 
   it("does an auth check on mount", () => {
-    render(<AuthProvider/>);
+    render(<AuthProvider />);
 
-    expect(mockAxios.get).toHaveBeenCalledWith("/api/me/");
+    expect(mockAxios.lastReqGet().url).toEqual("me/");
   });
 
-  it("doesn't crash if the initial check fails", () => {
-    const { getByText } = render(<AuthProvider>
-      test
-    </AuthProvider>);
+  it("doesn't crash if the initial check fails", async () => {
+    const { getByText } = render(<AuthProvider>test</AuthProvider>);
 
-    act(() => {
-      mockAxios.mockError(makeTestError(500, ERRORS.__TESTING));
+    await act(async () => {
```

Comment:
Just curious, but are the await-async acts necessary for the tests to run properly?

Reply:
If I don't do this, unfortunately I get weird, inconsistent behaviour. To be honest, I'm not completely sure what's going on.